model/artifact/db.go

```diff
@@ -9,15 +9,16 @@ import (
 
 var (
 	// BSON fields for artifact file structs
-	TaskIdKey     = bsonutil.MustHaveTag(Entry{}, "TaskId")
-	TaskNameKey   = bsonutil.MustHaveTag(Entry{}, "TaskDisplayName")
-	BuildIdKey    = bsonutil.MustHaveTag(Entry{}, "BuildId")
-	FilesKey      = bsonutil.MustHaveTag(Entry{}, "Files")
-	ExecutionKey  = bsonutil.MustHaveTag(Entry{}, "Execution")
-	CreateTimeKey = bsonutil.MustHaveTag(Entry{}, "CreateTime")
-	NameKey       = bsonutil.MustHaveTag(File{}, "Name")
-	LinkKey       = bsonutil.MustHaveTag(File{}, "Link")
-	AwsSecretKey  = "aws_secret"
+	TaskIdKey      = bsonutil.MustHaveTag(Entry{}, "TaskId")
+	TaskNameKey    = bsonutil.MustHaveTag(Entry{}, "TaskDisplayName")
+	BuildIdKey     = bsonutil.MustHaveTag(Entry{}, "BuildId")
+	FilesKey       = bsonutil.MustHaveTag(Entry{}, "Files")
+	ExecutionKey   = bsonutil.MustHaveTag(Entry{}, "Execution")
+	CreateTimeKey  = bsonutil.MustHaveTag(Entry{}, "CreateTime")
+	NameKey        = bsonutil.MustHaveTag(File{}, "Name")
+	LinkKey        = bsonutil.MustHaveTag(File{}, "Link")
+	ContentTypeKey = bsonutil.MustHaveTag(File{}, "ContentType")
+	AwsSecretKey   = "aws_secret"
```

Comment:
This isn't related to the change, but it's gonna bother me - can we change this to `bsonutil.MustHaveTag(File{}, "AwsSecret")`?